devicetypes/fibargroup/fibaro-wall-plug-eu-zw5.src/fibaro-wall-plug-eu-zw5.groovy

```diff
@@ -1,388 +1,388 @@
-}
-
-private parameterMap() {[
-		[key: "alwaysActive", num: 1, size: 1, type: "enum", options: [0: "function inactive", 1: "function activated"], def: "0", title: "Always On function",
-		 descr: "Turns the Wall Plug into a power and energy meter. Wall Plug will turn on connected device permanently and will stop reacting to attempts of turning it off."],
-		[key: "restoreState", num: 2, size: 1, type: "enum", options: [0: "device remains switched off", 1: "device restores the state"], def: "1", title: "Device restores the state before the power failure",
-		 descr: "After the power supply is back on, the Wall Plug can be restored to previous state or remain switched off."],
-		[key: "overloadSafety", num: 3, size: 2, type: "number", def: 0, min: 0, max: 30000 , title: "Overload safety switch",
-		 descr: "Allows to turn off the controlled device in case of exceeding the defined power; 1-3000 W.\n0 - function inactive\n10-30000 (1.0-3000.0W, step 0.1W)"],
-		[key: "standardPowerReports", num: 11, size: 1, type: "number", def: 15, min: 1, max: 100, title: "Standard power reports",
-		 descr: "This parameter determines the minimum percentage change in active power that will result in sending a power report.\n1-99 - power change in percent\n100 - reports are disabled"],
-		[key: "powerReportFrequency", num: 12, size: 2, type: "number", def: 30, min: 5, max: 600, title: "Power reporting interval",
-		 descr: "Time interval of sending at most 5 standard power reports.\n5-600 (in seconds)"],
-		[key: "periodicReports", num: 14, size: 2, type: "number", def: 3600, min: 0, max: 32400, title: "Periodic power and energy reports",
-		 descr: "Time period between independent reports.\n0 - periodic reports inactive\n5-32400 (in seconds)"],
-		[key: "ringColorOn", num: 41, size: 1, type: "enum", options: [
-				0: "Off",
-				1: "Load based - continuous",
-				2: "Load based - steps",
-				3: "White",
-				4: "Red",
-				5: "Green",
-				6: "Blue",
-				7: "Yellow",
-				8: "Cyan",
-				9: "Magenta"
-		], def: "1", title: "Ring LED color when ON", descr: "Ring LED colour when the device is ON."],
-		[key: "ringColorOff", num: 42, size: 1, type: "enum", options: [
-				0: "Off",
-				1: "Last measured power",
-				3: "White",
-				4: "Red",
-				5: "Green",
-				6: "Blue",
-				7: "Yellow",
-				8: "Cyan",
-				9: "Magenta"
-		], def: "0", title: "Ring LED color when OFF", descr: "Ring LED colour when the device is OFF."]
-]}
+/**
+ *	Fibaro Wall Plug ZW5
+ */
+metadata {
+	definition (name: "Fibaro Wall Plug EU ZW5", namespace: "FibarGroup", author: "Fibar Group", ocfDeviceType: "oic.d.smartplug") {
+		capability "Switch"
+		capability "Energy Meter"
+		capability "Power Meter"
+		capability "Configuration"
+		capability "Health Check"
+		capability "Refresh"
+
+		command "reset"
+		
+		fingerprint mfr: "010F", prod: "0602", model: "1001", deviceJoinName: "Fibaro Outlet" //Fibaro Wall Plug EU ZW5
+		fingerprint mfr: "010F", prod: "0602"
```

Comment:
Note that fingerprints that don't specify a join name will default to the name of the DTH. If this is something we want to avoid, it might be necessary to specify explicitly here.